src/app/fyle/my-view-report/my-view-report.page.ts

```
@@ -433,38 +435,46 @@
-          },
-        });
-      },
-    });
+      });
+  }
+
+  submitReport(): void {
+    this.submitReportLoader = true;
+    this.spenderReportsService
+      .submit(this.reportId)
+      .pipe(finalize(() => (this.submitReportLoader = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)